### DISCUSSION TITLE
DOCKGE environment variables to create admin user and password

### DISCUSSION THREAD

**Discussion by rchekhina** [louislam/dockge#542] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

HI,
Should it possible to have environment variable to create user / password ?

Best regards.

### ✔️ Solution

compose.yaml:

services:
  dockge:
    image: louislam/dockge:1
    restart: unless-stopped
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ./data:/app/data
      # Stacks Directory
      # ⚠️ READ IT CAREFULLY. If you did it wrong, your data could end up writing into a WRONG PATH.
      # ⚠️ 1. FULL path only. No relative path (MUST)
      # ⚠️ 2. Left Stacks Path === Right Stacks Path (MUST)
      - /home/ddeploy/deployments:/opt/stacks
    environment:
      # Tell Dockge where to find the stacks
      - DOCKGE_STACKS_DIR=/opt/stacks
      - USER_ADMIN=admin
      - USER_ADMIN_PASSWORD=XXXXXXX

**Comment by potvinp** [comment]
This would be amazing, it would help expedite installations via automation platforms such as Ansible, Terraform, etc

**Comment by barcar** [comment]
+1 for this too. It would really help with deploying the agent using Ansible.

**Comment by leinardi** [comment]
is there currently some alternative to create the admin user via automation platforms?

**Comment by stehajo** [comment]
+1, This would make my day!
